src/filterObjIndexed.js

```diff
@@ -0,0 +1,33 @@
+var _curry2 = require('./internal/_curry2');
+var _reduce = require('./internal/_reduce');
+var keys = require('./keys');
+
+
+/**
+ * An Object-specific version of `filter`. The predicate is applied to three
+ * arguments: *(value, key, obj)*. If only the value is significant, use
```

Comment:
How about using monospace for `(value, key, obj)`. I have a feeling it's going to look weird in docs like that.